config/src/config/network_config.rs

```diff
@@ -38,7 +38,8 @@ pub const PING_FAILURES_TOLERATED: u64 = 5;
 pub const CONNECTIVITY_CHECK_INTERVAL_MS: u64 = 5000;
 pub const MAX_CONCURRENT_NETWORK_REQS: usize = 100;
 pub const MAX_CONNECTION_DELAY_MS: u64 = 60_000; /* 1 minute */
-pub const MAX_FULLNODE_OUTBOUND_CONNECTIONS: usize = 3;
+/// Max default fullnode outbound connections is now 1 to decrease load on network
+pub const MAX_FULLNODE_OUTBOUND_CONNECTIONS: usize = 1;
```

Comment:
thank you! 😭

Reply:
Is this also configurable through fullnode config yaml file?

Reply:
Yes, it's that value.  The reason it's listed as `MAX_FULLNODE_CONNECTIONS` is it doesn't apply to validators

Reply:
nit: can we make this configurable, too? Once we get to mainnet, I can see us wanting to ensure we have more outbound connections for fullnodes. Especially when we start to worry about security.

Reply:
This is configurable